plan/planbuilder.go

```diff
@@ -453,18 +454,119 @@ func (b *planBuilder) buildSimple(node ast.StmtNode) Plan {
 	return &Simple{Statement: node}
 }
 
+func (b *planBuilder) getDefaultValue(col *table.Column) (*expression.Constant, error) {
+	if value, ok, err := table.GetColDefaultValue(b.ctx, col.ToInfo()); ok {
+		if err != nil {
+			return nil, errors.Trace(err)
+		}
+		return &expression.Constant{Value: value, RetType: &col.FieldType}, nil
+	}
+	return nil, errors.Errorf("default column not found - %s", col.Name.O)
+}
+
+func (b *planBuilder) findDefaultValue(cols []*table.Column, name *ast.ColumnName) (*expression.Constant, error) {
+	for _, col := range cols {
+		if col.Name.L == name.Name.L {
+			return b.getDefaultValue(col)
+		}
+	}
+	return nil, errors.Errorf("default column not found - %s", name.Name.O)
```

Comment:
Can't find column here rather than can't find default column value. This error massage should be different from line 464.